### DISCUSSION TITLE
Yarn re-patch package already patched

### DISCUSSION THREAD

**Discussion by finnmerlett** [yarnpkg/berry#4170] [Q&A]
I love `yarn patch`, a great built-in implementation of patch-package. But often I need to redo patches I created earlier, and update them with new alterations. At present, I have to run `yarn patch package-in-question`, then manually re-do every change, then add in the new changes.

Is there any way to apply the old patch to the temp folder created by `yarn patch` so this step is not necessary? Or is there any other way to update a yarn patch? Many thanks.

**Comment by filipef101** [comment]
You can. ie: `yarn patch "foo@patch:foo@npm%3A11.1.0#./.yarn/patches/foo-npm-11.1.0-044300d42a.patch::version=11.1.0&hash=9322f9&locator=frontend%40workspace%3A."`
you can do `yarn why foo` to find what to put in inside the quotes, make sure you have `package-name@` prefixed

Source: https://stackoverflow.com/a/69117750
